tests/syntax/test_modular.py

```diff
@@ -143,18 +188,18 @@ def test_sequential_composition():
                 do Sub(1)
                 do Sub(5)
         scenario Sub(x):
-            ego = Object at x @ 0
-            terminate after 1
+            ego = new Object at x @ 0
+            terminate after 1 seconds
```

Comment:
This should be `steps`: the test should not assume any particular default timestep. Ditto for some of the other tests.